<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwCamPrez</class>
 <widget class="QMainWindow" name="mwCamPrez">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CamPrez</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="vlRender">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hlParams">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="bRefreshInfo">
          <property name="text">
           <string>Refresh Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bParamRender">
          <property name="text">
           <string>Param Render</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlRenderPath">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lRenderPath">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Render Path: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leRenderPath">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlImagePath">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lImagePath">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Image Path: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leImagePath">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bOpen">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlImageName">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lImageName">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Image Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leImageName"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sbPadding">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="prefix">
           <string>Padding: </string>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbImageExt">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlImageSize">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lImageSize">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Image Size: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lWidth">
          <property name="text">
           <string>Width: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbWidth">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>12000</number>
          </property>
          <property name="value">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lHeight">
          <property name="text">
           <string>Height: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbHeight">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>12000</number>
          </property>
          <property name="value">
           <number>2000</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lByFrame">
          <property name="text">
           <string>By Frame: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbByFrame">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlResult">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lResult">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Result: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lResultVal">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="bRender">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Render Previz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTurn">
       <attribute name="title">
        <string>Turn</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="4" column="0">
         <widget class="QPushButton" name="bCreateCamTurn">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Create Camera Turn</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="hlTurnAxe">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="lTurnFrontAxe">
            <property name="text">
             <string>Front Axe:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbTurnX">
            <property name="text">
             <string>X</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgTurnFrontAxe</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbTurnY">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgTurnFrontAxe</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbTurnZ">
            <property name="text">
             <string>Z</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgTurnFrontAxe</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="cbTurnInvert">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Invert Axe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="hlTurnDuration">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="lTurnDuration">
            <property name="text">
             <string>Duration = </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbTurnDuration">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lTurnFrames">
            <property name="text">
             <string>(Frame)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="cbInvertRotate">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Invert Turn</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabQuadra">
       <attribute name="title">
        <string>Quadra</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgTurnFrontAxe"/>
 </buttongroups>
</ui>
